main/dhcp/APKBUILD

```diff
@@ -11,28 +11,20 @@ depends=
 pkgusers="dhcp"
 pkggroups="dhcp"
 makedepends="perl linux-headers file"
-install="dhcp.pre-install dhcp.pre-upgrade dhcp.post-upgrade"
+install="$pkgname.pre-install $pkgname.pre-upgrade $pkgname.post-upgrade"
 subpackages="$pkgname-doc $pkgname-dev $pkgname-dbg dhclient dhcrelay"
-source="http://ftp.isc.org/isc/dhcp/${_realver}/$pkgname-$_realver.tar.gz
+source="http://ftp.isc.org/isc/$pkgname/${_realver}/$pkgname-$_realver.tar.gz
+	dhclient-script-fix-bare-ip.patch
```

Comment:
I generally prefer to avoid using `$pkgname` in `source` because you may want copy paste the url to browser to find new/latest version. The `$pkgname` makes that harder and it does not really provide any benefit using a variable there.

At times we also have two versions of the same package (for example php5 and php7) and when that happens you cannot use pkgname variable in `builddir` and `source`

Reply:
There were already variables in use in source, so copy / paste directly won't work anyway.

`(. APKBUILD; echo "$source")`